### PR TITLE
Bugfix

### PR DESCRIPTION
**Pull request by minqian**
Bugfix for assertion 'itPtr >= start_ptr && itPtr + ITER_BYTES <= end_ptr' fialed;
Bugfix for error use for vgetq_lane_type and vextq_type;

### CODE REVIEW

**Review by tqltech — COMMENTED** [review]
Reviewed the code

**Review by rejohn — COMMENTED** [review]
review the code

**Comment by tqltech** [comment]
agree

**Review by rejohn — APPROVED** [review]

File: src/util/simd_arm.h [hunk 1]
```diff
@@ -355,9 +479,11 @@ static really_inline m128 palignr(m128 a, m128 b, int count) {
...
     count = count & 0xff;
     if (likely(count < 16)) {
-        result.vect_s8 = vextq_s8(b.vect_s8, a.vect_s8, count);
+        result = extbyte_m128(b, a, count);
     } else if (count < 32) {
-        result.vect_s8 = vextq_s8(a.vect_s8, vdupq_n_s8(0x0), count - 16);
+        m128 zeroVect;
+        zeroVect.vect_s8 = vdupq_n_s8(0x0);
+        result = extbyte_m128(a, zeroVect, count - 16);
     } else {
```
Comment (tjr1219) [inline src/util/simd_arm.h:487]: access

**Review by tjr1219 — COMMENTED** [review]
access

**Review by tjr1219 — APPROVED** [review]
access

**Pull request merged**